patterns/base.scss

```diff
@@ -19,13 +19,50 @@
 // Form Elements
 //////////////////////////////
 
+// Input
+@import 'base/input/sass/input';
+// Label
+@import 'base/label/sass/label';
+
 // Radio Button
 @import 'base/radio/sass/radio';
+// Color Input
```

Comment:
Ignore my comment. Just thought you were doing placeholder imports to avoid the conflicts we've been having with individual features. Which might not be a bad idea.. to include all of the future intended items in comments until we get to them. Sort of like a checklist while avoiding merge conflicts with developing each individually off of the develop branch